_data/simple-icons.json

```diff
@@ -3900,6 +3900,11 @@
             "hex": "EB001B",
             "source": "https://brand.mastercard.com/brandcenter/mastercard-brand-mark/downloads.html"
         },
+        {
+            "title": "mastercomfig",
+            "hex": "009688",
+            "source": "https://github.com/mastercomfig/mastercomfig.github.io/tree/develop/img"
```

Comment:
Would it be worth updating the link to the below? Just for the specific version.
```
https://github.com/mastercomfig/mastercomfig.github.io/blob/d910ce7e868a6ef32106e36996c3473d78da2ce3/img/mastercomfig_logo.svg
```

Reply:
@adamrusted Done